internal/host/stopinstallation.go

```diff
@@ -27,8 +26,8 @@ func NewStopInstallationCmd(log logrus.FieldLogger, instructionConfig Instructio
 func (h *stopInstallationCmd) GetStep(ctx context.Context, host *models.Host) (*models.Step, error) {
 	step := &models.Step{}
 	step.StepType = models.StepTypeExecute
-	step.Command = "/usr/bin/podman"
-	cmdArgs := []string{"stop", "--all"}
+
+	const stopAllCommand = "podman stop `podman ps --format '\"'{{.ID}} {{.Names}}'\"' | grep -v next-step-runner | awk '\"'{print \\$1}'\"'`"
```

Comment:
how do we know this will work?

Reply:
I tested the command locally, with all the crazy escaping, will test e2e in the morning.